test/JDBC/jdbc_schedule

```diff
@@ -128,6 +128,9 @@ ignore#!#case_insensitive_collation-before-13-5-vu-cleanup
 ignore#!#jira-BABEL-3504-upgrade-vu-prepare
 ignore#!#jira-BABEL-3504-upgrade-vu-verify
 ignore#!#jira-BABEL-3504-upgrade-vu-cleanup
+ignore#!#BABEL-2170-vu-prepare
```

Comment:
Why it needs to ignore in jdbc schedule test ?

Reply:
Xiaohui reviewed it last week and had some concerns about multi-db tests running in jdbcs tests. There was one test case in BABEL-2170-vu-verify which needs to run in multi-db setting. 
All the tests will be executed in upgrade tests now in multi-db mode. Since JDBC tests are in single-db, I added ignore here.